resources/views/front/community-events/_event.blade.php

```diff
@@ -13,7 +13,11 @@
                 {{ $model->name }}
             </h3>
         </a>
-        <div class="card-subtitle text-muted mb-2">{{ $model->start_at->isoFormat('MMMM D, Y') }} - {{ $model->end_at->isoFormat('MMMM D, Y') }}</div>
+        @if ($model->jury_id)
+            <div class="card-subtitle text-muted mb-2">{{ $model->start_at->isoFormat('MMMM D, Y') }} - {{ $model->end_at->isoFormat('MMMM D, Y') }} - {!! __('front/community-events.fields.jury', ['user' => link_to_route('users.profile', $model->jury->name, [$model->jury])]) !!}</div>
+        @else
+            <div class="card-subtitle text-muted mb-2">{{ $model->start_at->isoFormat('MMMM D, Y') }} - {{ $model->end_at->isoFormat('MMMM D, Y') }}</div>
+        @endif
```

Comment:
We can avoid copy-pasting by just wrapping the last part with an if statement :) and make it multi line instead of a huge line in the editor